src/main/scala/pricemigrationengine/services/CohortTable.scala

```diff
@@ -28,31 +32,60 @@ object CohortTable {
   def update(result: EstimationResult): ZIO[CohortTable, CohortUpdateFailure, Unit] =
     ZIO.accessM(_.get.update(result))
 
+  private implicit val cohortItemDeserialiser: DynamoDBDeserialiser[CohortItem] =
+    cohortItem =>
+      getStringFromResults(cohortItem, "subscriptionNumber")
+        .map(CohortItem.apply)
+
+  private implicit val estimationResultSerialiser: DynamoDBUpdateSerialiser[EstimationResult] =
+    estimationResult =>
+      Map(
+        stringFieldUpdate("processingStage", EstimationComplete.value),
+        dateFieldUpdate("expectedStartDate", estimationResult.expectedStartDate),
+        doubleFieldUpdate("estimatedNewPrice", estimationResult.estimatedNewPrice)
+      ).asJava
```

Comment:
There are a couple of new fields now as well in the `estimationResult`: `currency` and `oldPrice`.